imhelping.cs

```diff
@@ -0,0 +1,20 @@
+public static class IMHelping {
+
+	public enum Direction
+	{
+		Forwards,
+		Backwards,
```

Comment:
Need comments to explain these concepts.

Reply:
This is a CLASS not an interface. what's the M stand for?  Use your words.

Reply:
OBVIOUSLY I was referring to the note about pascal case. I was so overcome with horror reading your code that I couldn't even click back on that section.